JetNews/app/src/main/java/com/example/jetnews/ui/Navigation.kt

```diff
@@ -0,0 +1,124 @@
+package com.example.jetnews.ui
+
+import android.os.Bundle
+import androidx.annotation.MainThread
+import androidx.compose.getValue
+import androidx.compose.setValue
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import com.example.jetnews.ui.Screen.Article
+import com.example.jetnews.ui.Screen.Home
+import com.example.jetnews.ui.Screen.Interests
+import com.example.jetnews.ui.ScreenNames.ARTICLE
+import com.example.jetnews.ui.ScreenNames.HOME
+import com.example.jetnews.ui.ScreenNames.INTERESTS
+import com.example.jetnews.utils.getMutableStateOf
+
+/**
+ * Screen names (used for serialization)
+ */
+enum class ScreenNames { HOME, INTERESTS, ARTICLE }
```

Comment:
Nit: Maybe `ScreenName`? It's weird that in `Screen` (below) it takes an `ScreenNames` param

Reply:
Doing this – also if you can think of a way to remove this enum completely very happy to do so (it seems a bit repetitive to me)

Reply:
It is a bit repetitive but if we want to serialize the screens, not sure there's another way :) I'm ok with having it there